### PR TITLE
Add error message when you try to download an image in a folder with no write permission

### PR DESCRIPTION
**Pull request by IrvinLara9**
Solves #2474 (Saving a file does not report failures)

**Description**
Now when you try to download an image in a folder where you don't have write permission, an error alert is displayed to the user, indicating the problem.

**Why**
When the error of the write permission occurred, the app didn't notify the user that an error occurred.

**Additional information**

- Photo of the current error message:
Screen Shot 2021-12-28 at 20 14 50

### CODE REVIEW

File: src/main/ipc.ts [hunk 1]
```diff
@@ -132,7 +132,16 @@ export async function init(cwd: string, logHandler: LogHandler) {
...
-      await copyFile(source, filePath)
+      try {
+        await copyFile(source, filePath)
+      } catch (error) {
+        if (error.code == 'EACCES') {
+          dialog.showErrorBox(
+            'Permission Error',
+            `Cannot write in this folder. You don't have write permission`
+          )
+        }
```
Comment (Jikstra) [inline src/main/ipc.ts:143]: Can you also handle the else case? Maybe something as 
```
} else {
          dialog.showErrorBox(
            'Unhandled error',
            `Cannot copy file. Error: ${error}`
          )
}
```

**Comment by Jikstra** [comment]
Thank you! Looks good :) If you have time & motivation it would be nice to also have the else case handled. But already like this I would be happy to merge this pr :)

**Comment by IrvinLara9** [comment]
Hi @Jikstra Thank you for the comment! I have already added your suggestion!

**Comment by Jikstra** [comment]
Perfect, thank you!

**Pull request merged**